pkg/module/parse/template_to_module.go

```diff
@@ -180,9 +180,8 @@ func insertLayerIntoManifest(
 	manifest *v1beta2.Manifest, layer img.Layer,
 ) error {
 	switch layer.LayerName {
+	case img.ConfigLayer: // Skipping Config Layer. Deprecated since removal of Helm and Kustomize from ModuleTemplates
```

Comment:
Just skip is not enough, you need to remove the usage of `manifest.Spec.Config` code completely.